### DISCUSSION TITLE
What are the different ways in which I can launch/Install Ant Media server?

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5288] [FAQs]
I would like to explore the various methods available for installing and launching Ant Media Server.

Can someone please provide a list of the different options for installing and launching AMS, both for on-premise installations and cloud-based deployments? I would like to compare these options and choose the best one that suits my needs.

**Comment by umtcylmz** [comment]
Hi Mohit,

AMS can be deployed easily and quickly on-premises or on public cloud networks such as [AWS](https://aws.amazon.com/marketplace/pp/prodview-464ritgzkzod6?sr=0-1&ref_=beagle&applicationId=AWSMPContessa&_ga=2.27383807.275147218.1658729962-339983394.1658135231#pdp-pricing), [Azure](https://azuremarketplace.microsoft.com/en/marketplace/apps/antmedia.ant_media_server_enterprise?tab=Overview), Google Cloud Platform, [Linode](https://www.linode.com/marketplace/apps/ant-media/ant-media-enterprise-edition/), [Alibaba](https://marketplace.alibabacloud.com/store/3246141.html), [Digital Ocean](https://marketplace.digitalocean.com/apps/ant-media-server-enterprise-edition) or OVH.  

For the on-premise installation, you can directly visit (https://antmedia.io/) and subscribe to Monthly, Annual, or Perpetual Licenses. You'll receive the license key from contact@antmedia.io email address. You can follow these [instructions](https://antmedia.io/docs/guides/installing-on-linux/installing-ams-on-linux/) for a quick start.
